.dockerignore

```diff
@@ -0,0 +1,5 @@
+./examples
+./target
+LICENSE
```

Comment:
Might need to keep the `LICENSE` file in the packaging...

Reply:
Ok. Done in https://github.com/CanalTP/transit_model/pull/617/commits/91c57e7a5d2e985469bf019544df22dbfc32a174